open-metadata-implementation/access-services/digital-service/digital-service-server/README.md

```diff
@@ -0,0 +1,28 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+![InDev](../../../open-metadata-publication/website/images/egeria-content-status-in-development.png#pagewidth)
+
+# DevOps Open Metadata Access Service (OMAS)
```

Comment:
DevOps?

Also - this should be about the server - not about all of the modules.

Reply:
resolved

Reply:
resolved